config/schedule.rb

```diff
@@ -0,0 +1,26 @@
+# Use this file to easily define all of your cron jobs.
```

Comment:
Let's actually move the use of this gem. In production (on Heroku), we have a task scheduler.